apps/sms/views/conversation/js/conversation.js

```diff
@@ -763,15 +767,14 @@ var ConversationView = {
           }
 
           this.recipients.add(recipient);
-        }.bind(this));
-      }, this);
+
+          this.draft.isEdited = false;
+        });
+      });
 
       // Render draft contents into the composer input area.
       Compose.fromDraft(this.draft);
 
-      // Discard this draft object and update the backing store
-      Drafts.delete(this.draft).store();
```

Comment:
> Now I think we need some new logic to update a draft, keeping its id. 

Mm, I think we already somewhat have it - when we call ```Drafts.add``` with the draft id that is already exists - we remove old draft and re-create one with the same id.

> For example, because we don't have the "reject call then send message" feature, we could imagine that a user saves a draft "I am in a meeting" to be easily recalled.

Heh, yeah, good use case. But I would say - that it's not a draft, but a feature of Dialer - when Dialer provides a list of possible messages (maybe with smart heuristics depending on who is calling, what time of the day or what current phone mode is and etc.) and then uses "new" messages activity - so that they are easily and quickly accessible in one compact list in the context of Dialer. But bookmarking sounds as a good idea as well, I could imagine user can pin draft so that it's permanently pinned and doesn't disappear when sent.